packages/nc-gui/components/virtual-cell/Lookup.vue

```diff
@@ -46,45 +46,87 @@ provide(MetaInj, lookupTableMeta)
 provide(CellUrlDisableOverlayInj, ref(true))
 
 const lookupColumnMetaProps = useColumn(lookupColumn)
+
+const showEditWarning = ref(false)
+
+const showClearWarning = ref(false)
+
+const showEditWarningMessage = () => {
+  showEditWarning.value = true
+
+  setTimeout(() => {
+    showEditWarning.value = false
+  }, 3000)
+}
+const showClearWarningMessage = () => {
+  showClearWarning.value = true
+
+  setTimeout(() => {
+    showClearWarning.value = false
+  }, 3000)
+}
+
+useSelectedCellKeyupListener(inject(ActiveCellInj, ref(false)), (e: KeyboardEvent) => {
+  switch (e.key) {
+    case 'Enter':
+      showEditWarningMessage()
```

Comment:
Same here: use [`refAutoReset`](https://vueuse.org/shared/refAutoReset/) to save yourself some lines of code.